src/components/SponsorButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SponsorButton = ({email, subject}) => (
+  <a href={`mailto:${email}?subject=${subject}`} className="sponsor">
+    Seja Patrocinador
+  </a>
+);
+
+SponsorButton.propTypes = {
+  email: PropTypes.string.isRequired,
+  subject: PropTypes.string,
+}
+
+SponsorButton.defaultProps = {
+  subject: "Interesse em ser patrocinador",
```

Comment:
Since you are using a function component, you can specify this in the function itself.

Reply:
Explain me this, please :pray:

Reply:
Okay :+1: